data-upgrade-users/pom.xml

```diff
@@ -0,0 +1,74 @@
+<!--
+
+    Copyright (C) 2022 eXo Platform SAS.
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Affero General Public License for more details.
+    You should have received a copy of the GNU Affero General Public License
+    along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.exoplatform.addons.upgrade</groupId>
+    <artifactId>upgrade</artifactId>
+    <version>6.3.x-maintenance-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>data-upgrade-users</artifactId>
+  <packaging>jar</packaging>
+  <name>eXo Add-on:: Data Upgrade Add-on - users</name>
+  <description>A module to migrate lastLoginTime field on 6.1.x to 6.3.x </description>
+  <properties>
+    <exo.test.coverage.ratio>0</exo.test.coverage.ratio>
```

Comment:
can you update the coverage ratio ? as you made unit test you should be able to put it higher than 0